composer.lock

```diff
@@ -15638,5 +15629,5 @@
         "composer-runtime-api": "^2.1"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
Please check you composer version and if it is lower than 2.4.0, please run composer self-update. Every time when you update something on composer, it downgrades `plugin-api-version`.

Reply:
It's because used not sdk docker container for the development. @vol4onok, please reinstall composer dependencies using sdk docker container to avoid such updates.

Reply:
composer updated it. I just use the released version for spryk.

Reply:
what version of composer itself do you use? 

```
composer --version
```

Reply:
Composer version 2.2.1

Reply:
ok. anyway, I don't think it is an issue for us

Reply:
we have direct and indirect dependencies. we should fix indirect dependencies because the composer checks and monitors that and according to composer,json

Reply:
Please just execute locally

```
composer self-update
```

After that run ```composer update spryker-sdk/spryk```

Reply:
`Composer version 2.3.10 2022-07-13 15:48:23` in the `sdk` container. Afaik we agreed to use sdk container for the development. It is just a recommendation to avoid such cases like this one. I don't insist.
On the flip side, I believe what @vol4onok wants to say is that it does not always depend only on the composer version https://getcomposer.org/doc/articles/composer-platform-dependencies.md#plugin-package-composer-plugin-api.